scriptmodules/emulators/jzintv.sh

```diff
@@ -31,24 +33,44 @@ function sources_jzintv() {
 function build_jzintv() {
     mkdir -p jzintv/bin
     cd jzintv/src
+
+    # Add source release date information to build
+    mv buildcfg/90-svn.mak buildcfg/90-svn.mak.txt
+    echo "SVN_REV := $(echo $md_repo_url | grep -o -P '[\d]{8}')" > buildcfg/90-src_releasedate.mak
+    sed -i.zip-dist "s/SVN Revision/Releasedate/" svn_revision.c
+
     make clean
     make CC="gcc" CXX="g++" WARN="" WARNXX="" OPT_FLAGS="$CFLAGS"
+
     md_ret_require="$md_build/jzintv/bin/jzintv"
 }
 
 function install_jzintv() {
+    rm -f jzintv/bin/*.dll
```

Comment:
This should also be moved to the `sources` function. Since the folder is created in `build`, you could remove it entirely.

Reply:
Addressed in update 17bae33

Reply:
Addressed in update 17bae33.